single-page.html

```diff
@@ -1467,7 +1464,10 @@ <h2 class="no-num no-toc no-ref heading settled" id="status"><span class="conten
     technical report can be found in the <a href="https://www.w3.org/TR/"><abbr title="World Wide Web Consortium">W3C</abbr> technical
     reports index</a> at https://www.w3.org/TR/.</em> </p>
 
-<p><a href="https://github.com/w3c/html/labels/errata">Errata</a> for this document are recorded as Github issues with the "errata" label.The <a href="https://w3c.github.io/html/">latest HTML editors' draft</a> shows proposed resolutions of errata <i>in situ</i>.</p>
+
+   <p>This specification defines the 2nd edition of the 5th major version, first minor revision of the core language of the World Wide Web: the Hypertext Markup Language (HTML). In this version, a significant error with <a href="https://w3.org/tr/51/">HTML 5.1</a> has been rectified, as noted in the <a href="errata">Errata corrected</a> section.</p>
+
+<p><a href="https://github.com/w3c/html/labels/errata">Errata</a> for this document are recorded as Github issues with the "errata" label. The <a href="https://w3c.github.io/html/">latest HTML editors' draft</a> shows proposed resolutions of errata <i>in situ</i>.</p>
```

Comment:
Should we track errata by project, so we would set up a project for any spec that has an erratum, as well as the label?

I hope we don't have a lot of errata, but I can imagine 1 or 2 per year meaning we have a page-long list, and will want to make it clear that most of them aren't what someone is looking for - assuming that is mostly errata for the then-current version…